src/flash/nor/stm32f1x.c

```diff
@@ -1751,3 +1751,55 @@ const struct flash_driver stm32f1x_flash = {
 	.info = get_stm32x_info,
 	.free_driver_priv = default_flash_free_driver_priv,
 };
+
+/* flash bank gd32vf103 <base> <size> 0 0 <target#>
+ */
+FLASH_BANK_COMMAND_HANDLER(gd32vf103_flash_bank_command)
+{
+	struct stm32x_flash_bank *stm32x_info;
+
+	LOG_WARNING("The gd32vf103 flash target will be removed in June of 2023, please use stm32f1x instead.");
```

Comment:
Here I'd suggest a minor change to improve the visibility of the message:

```suggestion
	LOG_WARNING("DEPRECATED: The gd32vf103 flash target will be removed in June of 2023, please use stm32f1x instead.");
```

Reply:
Fixed